go/common/sgx/ias/quote.go

```diff
@@ -146,6 +174,19 @@ func (q *Quote) Verify() error {
 	if mrSignerBlacklist[q.Report.MRSIGNER] {
 		return fmt.Errorf("ias/quote: blacklisted MRSIGNER")
 	}
+
+	if !unsafeAllowDebugEnclaves {
+		// Drop debug enclaves, if we are in production mode.
```

Comment:
Maybe instead of "drop" use the term "disallow" (everywhere, also in error message below).